resources/migrations/001_update_migrations.yaml

```diff
@@ -8253,6 +8253,152 @@ databaseChangeLog:
             tableName: login_history
             indexName: idx_user_id_device_id
 
+  - changeSet:
+      id: v51.2024-06-13T11:22:39
+      author: johnswanson
+      comment: Add `last_viewed_at` to dashboards
+      changes:
+        - addColumn:
+            tableName: report_dashboard
+            columns:
+              - column:
+                  name: last_viewed_at
+                  type: ${timestamp_type}
+                  remarks: Timestamp of when this dashboard was last viewed
+                  constraints:
+                    nullable: true
+
+  - changeSet:
+      id: v51.2024-06-14T06:45:34
+      author: johnswanson
+      comment: Create index on view_log (model, model_id, timestamp)
+      changes:
+        - createIndex:
+            tableName: view_log
+            columns:
+              - column:
+                  name: model
+              - column:
+                  name: model_id
+              - column:
+                  name: timestamp
+            indexName: idx_view_log_model_model_id_timestamp
+
+  - changeSet:
+      id: v51.2024-06-14T06:45:35
+      author: johnswanson
+      comment: Populate `last_viewed_at` for dashboards
+      rollback: # not needed, will be removed
+      changes:
```

Comment:
maybe you already know this but just want to share an interesting finding I found recently https://metaboat.slack.com/archives/C0641E4PB9B/p1720015474515889

Reply:
Lol, yes - I also discovered this incredibly fun fact. I love writing migrations like this for 4 separate database types 😅